homeassistant/components/modbus/modbus.py

```diff
@@ -275,8 +275,26 @@ def __init__(self, hass: HomeAssistant, client_config: dict[str, Any]) -> None:
                     "url": "https://www.home-assistant.io/integrations/modbus",
                 },
             )
-            _LOGGER.warning(
-                "`close_comm_on_error`: is deprecated and will be remove in version 2024.4"
+            _LOGGER.warning(  # pragma: no cover
```

Comment:
I think this no cover is not needed anymore if I remember correctly

Reply:
Ups the PR reintroduced a handful, being removed.

Reply:
Done.